src/answers-umd.js

```diff
@@ -588,6 +606,25 @@ class Answers {
   _getInitLocale () {
     return this.core.storage.get(StorageKeys.LOCALE);
   }
+
+  /**
+   * Parses a value from persistent storage, which stores strings,
```

Comment:
What do you think about having this be its own class? I was envisioning we'd have a class some day whose responsibility is to properly seed `storage` with whatever is in the URL. Out of scope for this PR, but certain components have parsing logic like this. With such a class, we could move that parsing out of the components.

Reply:
Yeah I think that makes sense!  is it okay if I leave it here for this pr?

Reply:
Sure, but let's make a TODO.

Reply:
added a todo!